cmd/src/scout_usage.go

```diff
@@ -92,14 +96,17 @@ func init() {
 		if *pod != "" {
 			options = append(options, usage.WithPod(*pod))
 		}
+		if *container != "" || *docker {
+			if *container != "" {
+				options = append(options, usage.WithContainer(*container))
+			}
```

Comment:
is it intentional that setting container always also flips on the docker flag?

Reply:
That was my intention. For kubernetes I have the --pod flag.

Reply:
Essentially I wanted --container to be associated with Docker deployments and --pod with Kubernetes. So:

`src scout usage --container gitserver-0`
`src scout usage --container gitserver-0 --docker`
`src scout usage --docker --container gitserver-0`

Would all have the same output.